pkg/notify/notify.go

```diff
@@ -171,11 +171,11 @@ func (r *Receiver) Notify(data *alertmanager.Data, hashJiraLabel bool, updateSum
 	}
 
 	if len(data.Alerts.Firing()) == 0 {
-		level.Debug(r.logger).Log("msg", "no firing alert; nothing to do.", "label", issueGroupLabel)
+		level.Debug(r.logger).Log("msg", "no firing alert; nothing to do.", r.conf.FieldLabels, issueGroupLabel)
```

Comment:
Not sure if we should use `IDLabel` value for logging key. What if somebody will have custom field called `msg`? Can we just use `id` here? 

Also I would rename `issueGroupLabel` everywhere to `issueIDLabel` because at this point it might be hash, static and maybe group labels. "Group" is a bit misleading given the `add_group_labels` config field which only referes to Prometheus labels. 🙈 not an ideal name, but not worth changing (unless we want to deprecate)